src/main/java/io/opentracing/contrib/spanmanager/SpanManager.java

```diff
@@ -48,7 +48,7 @@
     /**
      * Return the current {@link ManagedSpan}.
      *
-     * @return The current ManagedSpan or the managed <code>NoopSpan</code> if there is no managed span.
+     * @return The current ManagedSpan or null if there is no managed span.
```

Comment:
I think I misunderstood.
For semantics, I think we can return an EmptyManagedSpan / NoManagedSpan constant here, that contains a `null` Span.
That way for Java-8, we could add `Optional`-like methods to the interface providing a `getOrElse(Span)` or `ifPresent(Consumer<Span>)` default method.

I thought you would change the getSpan() to return `null`.